.github/workflows/build-images-ci.yaml

```diff
@@ -153,7 +153,7 @@ jobs:
           tags: |
             quay.io/${{ github.repository_owner }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}-race
           build-args: |
-            BASE_IMAGE=quay.io/cilium/cilium-runtime:28b5a8658b596d12d80b0e7dad3efc2e77ec2d65@sha256:fbf8eee141101fade247dbe94cf84ca3bdcd92b96108996c50859ab7edd607d0
+            BASE_IMAGE=quay.io/cilium/cilium-runtime:6e2e3f2606e478b1c9a404070e3b21f2f5496445
```

Comment:
This GitHub workflow runs on `pull_request_target`. You will have to open a temporary, draft PR with that trigger changed to `pull_request` to properly test this pull request. Otherwise, it will run the master version of this workflow.

Reply:
@pchaigno I've opened #16543. Can you have a look and tell if that seems correct?

Reply:
Looks good. I'll trigger the tests.

Reply:
I forget that won't work with a forked repository :facepalm: I'll copy your branch and will send it on my side :-)

Reply:
Testing at https://github.com/cilium/cilium/pull/16553.

Reply:
The tests at #16553 are currently failing because of the missing SHA256 hashes. We'll have to rerun once those are updated.